kolibri/plugins/learn/assets/src/views/ContentCardListViewItem.vue

```diff
@@ -0,0 +1,318 @@
+<template>
+
+  <router-link
+    :to="link"
+    class="card"
+    :class="[
+      { 'mobile-card': isMobile },
+      $computedClass({ ':focus': $coreOutline })
+    ]"
+    :style="{ backgroundColor: $themeTokens.surface }"
+  >
+
+    <div class="thumbnail">
+      <CardThumbnail
+        v-bind="{ thumbnail, kind, isMobile }"
+      />
+      <KLinearLoader
+        class="k-linear-loader"
+        :delay="false"
+        :progress="progress"
+        type="determinate"
+        :style="{ backgroundColor: $themeTokens.fineLine }"
+      />
+    </div>
+    <span class="details" :style="{ color: $themeTokens.text }">
+      <div class="metadata-info">
+        <KLabeledIcon
+          :icon="`${kindToLearningActivity}Solid`"
+          size="mini"
+          :label="learnString(kindToLearningActivity)"
+        />
+      </div>
+      <h3 class="title" dir="auto">
+        <TextTruncator
+          :text="title"
+          :maxHeight="maxTitleHeight"
+        />
+      </h3>
+      <p class="text">
+        <TextTruncator
+          :text="description"
+          :maxHeight="maxDescriptionHeight"
+        />
+      </p>
+      <div class="metadata-info">
+        <p> {{ displayCategoryAndLevelMetadata }}</p>
+      </div>
+      <img
+        :src="channelThumbnail"
+        :alt="$tr('logo', { channelTitle: channelTitle })"
```

Comment:
Same concern as above, if the thumbnail is adjacent to the visible string with channel title, the image should be marked as decorative.